src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -28,6 +28,9 @@
 // - AnnounceOTAProviders command is ignored if an update is in progress
 // - The provider location passed in AnnounceOTAProviders is used in a single query (possibly retried) and then discarded
 // - Explicitly triggering a query through TriggerImmediateQuery() cancels any in-progress update
+// - A QueryImage call results in the driver iterating through the list of default OTA providers, from beginning to the end, until a
+//   provider successfully transfers the OTA image. If a provider is busy, it will be retried a set number of times before moving
+//   to the next available one. If all else fails, the default provider timer is kicked off again.
```

Comment:
nit: If there is another iteration of commit, I'd update the text to indicate `default provider timer` == `periodic query timer`

Reply:
Done